tests/func/test_api.py

```diff
@@ -62,29 +61,20 @@ def test_open(tmp_dir, dvc, remote):
 @pytest.mark.parametrize(
     "cloud",
     [
-        *[
-            pytest.lazy_fixture(cloud)
-            for cloud in [
-                "real_s3",  # NOTE: moto's s3 fails in some tests
-                "gs",
-                "azure",
-                "gdrive",
-                "oss",
-                "http",
-                "hdfs",
-            ]
-        ],
-        pytest.param(
-            pytest.lazy_fixture("ssh"),
-            marks=pytest.mark.xfail(
-                raises=DvcException,
-                condition=platform.system() == "Windows",
-                reason="https://github.com/iterative/dvc/issues/4418",
-            ),
-        ),
+        pytest.lazy_fixture(cloud)
+        for cloud in [
+            "real_s3",  # NOTE: moto's s3 fails in some tests
+            "gs",
+            "azure",
+            "gdrive",
+            "oss",
+            "http",
+            "hdfs",
+            "ssh",
+        ]
     ],
 )
-def test_open_external(erepo_dir, cloud):
+def test_open_external(tmp_dir, erepo_dir, cloud):
```

Comment:
Why is `tmp_dir` needed now?

Reply:
Github actions clones the repository on `D`, so SSH server is initialized in a different drive than our storage. In order to be able to use SSH, we need to use Posix  style paths, and during SSH initialization we omit the drive information. This results in a situation, where we cannot access our storage. Added check which will let developers know about such situation.

Reply:
Oh, great catch! :fire:  I remember having some ssh tests like these too, could you please grep the tests to see if there are any ssh tests like that?

Also seems like using `tmp_dir` here is still more of a hack, because the only reason we use it is for ssh to be initialized properly. Maybe we could alter the ssh fixture to start sftp where we want it (on C i suppose)?